Test/Utils/ComputeNode/Service.ps1

```diff
@@ -0,0 +1,93 @@
+. $PSScriptRoot\..\..\PesterLogger\PesterLogger.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Common\Invoke-NativeCommand.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testbed.ps1
+. $PSScriptRoot\Configuration.ps1
+
+function Install-ServiceWithNSSM {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName,
+        [Parameter(Mandatory=$true)] $ExecutablePath,
+        [Parameter(Mandatory=$false)] [string[]] $CommandLineParams = @()
+    )
+
+    $Output = Invoke-NativeCommand -Session $Session -ScriptBlock {
+            nssm install $using:ServiceName "$using:ExecutablePath" $using:CommandLineParams
+    } -AllowNonZero -CaptureOutput
+
+    $NSSMServiceAlreadyCreatedError = 5
+    if ($Output.ExitCode -eq 0) {
+        Write-Log $Output.Output
+    }
+    elseif ($Output.ExitCode -eq $NSSMServiceAlreadyCreatedError) {
+        Write-Log "$ServiceName service already created, continuing..."
+    }
+    else {
+        throw [HardError]::new("Unknown (wild) error appeared while creating $ServiceName service")
+    }
+}
+
+function Remove-ServiceWithNSSM {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName
+    )
+
+    $Output = Invoke-NativeCommand -Session $Session {
+        nssm remove $using:ServiceName confirm
+    } -CaptureOutput
+
+    Write-Log $Output
+}
+
+function Enable-Service {
```

Comment:
Enable/Disable is not a good name in my opinion. E.g. on systemd Enable/disable is used to signify if service is started automatically.

Maybe `Enable-Service` should be named `Start-RemoteService`. The same applies to `Disable-Service`.